src/engine/renderer/tr_surface.cpp

```diff
@@ -550,12 +558,15 @@ void Tess_InstantQuad( u_ModelViewProjectionMatrix &shader, const float x, const
 
 	Tess_Begin( Tess_StageIteratorDummy, nullptr, true, -1, 0 );
 
+	/* These values make possible to use either the legacy quad rendering
```

Comment:
The point of this function is to render any screen-space axis-aligned rectangle specified by `x, y, width, height` so the comment seems out of place.